tfjs-backend-webgpu/src/kernels/DepthToSpace.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {DepthToSpace, DepthToSpaceAttrs, DepthToSpaceInputs, KernelConfig, KernelFunc, TensorInfo, util} from '@tensorflow/tfjs-core';
+
+import {WebGPUBackend} from '../backend_webgpu';
+import {DepthToSpaceProgram} from './depth_to_space_webgpu';
+
+export function depthToSpace(args: {
+  inputs: DepthToSpaceInputs,
+  backend: WebGPUBackend,
+  attrs: DepthToSpaceAttrs
+}): TensorInfo {
+  const {inputs, backend, attrs} = args;
+  const {x} = inputs;
+  const {blockSize, dataFormat} = attrs;
+
+  util.assert(
```

Comment:
It's better to move the checking to tfjs-core https://github.com/tensorflow/tfjs/blob/master/tfjs-core/src/ops/depth_to_space.ts#L92

Reply:
Done.